amaranth-chrome-ext/src/AmaranthUtil.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Utility class with helpful functions for the Amaranth Chrome extension.
```

Comment:
s/Utility class/Utilities

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/a95b2263dbb2ca0b10330077196a4696d6f03db2